requirements.txt

```diff
@@ -1,2 +1,4 @@
 Django==2.2.4
 psycopg2-binary==2.8.3
+selenium == 3.8
```

Comment:
spacje są raczej nie wskazane pomiędzy ==
wynika to raczej z konwencji, że często ten plik jest generowany przez komendę `pip freeze` i taki format akurat przyjmuje